strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFileTests/Tests.cs

```diff
@@ -0,0 +1,34 @@
+using ReadStringFromResourceFile;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests
+{
+    [TestClass]
+    public class Tests 
+    {
+        ResourcesManager rmEnglish;
+        ResourcesManager rmPortuguese;
+
+        public Tests()
+        {
+            rmEnglish = new ResourcesManager(@"ReadStringFromResourceFile.Resources.Texts.English");
+            rmPortuguese = new ResourcesManager(@"ReadStringFromResourceFile.Resources.Texts.Portuguese");
+        }
```

Comment:
These are not following naming conventions. Also, this can be simplified by converting these statements into member initializers and removing the constructor altogether